File: rust/rpcpool/tests/solana-rpc-test.ts
import { Connection, PublicKey } from "@solana/web3.js";

(async () => {
  const connection = new Connection("http://0.0.0.0:8000");
  const address = new PublicKey("GQFJibdqFGdNXm5PKvm4MmEDgNXHP1JcXNonedi2Z7kT");
  const balance = await connection.getBalance(address);
  console.log(balance);
})();

File: rust/rpcpool/tests/solana-rpc-test.rs
use solana_client::rpc_request::Address;

#[test]
fn test_solana_rpc_pool_proxy() {
    let address = Address::from_str_const("GQFJibdqFGdNXm5PKvm4MmEDgNXHP1JcXNonedi2Z7kT");
    let rpc = solana_client::rpc_client::RpcClient::new("https://rpcpool.rhiva.fun/");
    let balance = rpc.get_balance(&address).unwrap();
    print!("balance={}", balance);
}
